src/routes/activity/ActivityStyle.js

```diff
@@ -1,4 +1,203 @@
 import styled from 'styled-components';
 import palette from '@styles/theme';
 
-export const Activity = styled.div``;
+export const Activity = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+`;
+
+export const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 50px;
+  padding: 70px 80px;
+
+  /* ≤1023px (태블릿) */
+  @media (max-width: 1023px) {
+    gap: 40px;
+    padding: 56px 40px;
+  }
+
+  /* ≤767px (모바일) */
+  @media (max-width: 767px) {
+    gap: 32px;
+    padding: 30px 20px;
+  }
+`;
+
+export const HeaderName = styled.h1`
+  color: ${palette.mainNavy.navy100};
+  font-size: 36px;
+  font-weight: 600;
+  margin: 0;
+
+  @media (max-width: 1023px) {
+    font-size: 30px;
+  }
+  @media (max-width: 767px) {
+    font-size: 24px;
+  }
+`;
+
+export const ToggleBox = styled.div`
+  width: 100%;
+`;
+
+export const ToggleName = styled.div`
+  color: ${palette.mainNavy.navy80};
+  font-size: 24px;
+  font-weight: 600;
+  margin-bottom: 25px;
+  display: flex;
+  align-items: center;
+
+  img {
+    margin-right: 5px;
+  }
+
+  @media (max-width: 1023px) {
+    font-size: 20px;
+    margin-bottom: 20px;
+  }
+  @media (max-width: 767px) {
+    font-size: 18px;
+    margin-bottom: 16px;
+  }
+`;
+
+export const BranchContainer = styled.div`
+  display: flex;
+  gap: 96px;
+  padding-top: 25px;
+  align-items: flex-start;
+
+  @media (max-width: 1023px) {
+    gap: 48px;
+    padding-top: 20px;
+  }
+  /* 모바일에선 세로 스택 */
+  @media (max-width: 767px) {
+    flex-direction: column;
+    gap: 24px;
+    padding-top: 16px;
+  }
+`;
+
+/* 분기 별 컴포넌트 */
+export const BungiContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  row-gap: 25px;
+
+  @media (max-width: 1023px) {
+    row-gap: 20px;
+  }
+  @media (max-width: 767px) {
+    row-gap: 16px;
+  }
+`;
+
+export const BContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+export const BHeaderBox = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 5px;
+  margin-bottom: 15px;
+
+  @media (max-width: 767px) {
+    margin-bottom: 10px;
+  }
+`;
+
+export const BHeader = styled.div`
+  color: ${palette.text.primary};
+  font-size: 20px;
+  font-weight: 600;
+
+  @media (max-width: 1023px) {
+    font-size: 18px;
+  }
+  @media (max-width: 767px) {
+    font-size: 16px;
+  }
+`;
+
+export const BMonth = styled.div`
+  color: ${palette.text.secondary};
+  font-size: 14px;
+  font-weight: 400;
+
+  @media (max-width: 767px) {
+    font-size: 13px;
+  }
+`;
+
+export const ListContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 20px 15px;
+  row-gap: 10px;
+  border-radius: 17px;
+  border: 1px solid ${palette.hover.back2};
+
+  /* 고정폭 → 반응형 */
+  width: clamp(280px, 92vw, 685px);
+  margin: 0 auto;
+
```

Comment:
```suggestion
  transition: 300ms ease;
  &:hover {
    box-shadow: 0 0 15px 0 ${palette.mainNavy.navy10};
  }
  
```

  피그마에 `각 카드 호버시 그림자 추가`가 있어서 해당 코드 추가해주시면 좋을 것 같습니다!